deon/assets/examples_of_ethical_issues.yml

```diff
@@ -16,6 +16,14 @@
       url: https://www.theguardian.com/technology/2014/jun/27/new-york-taxi-details-anonymised-data-researchers-warn
     - text: Netflix prize dataset of movie rankings by 500,000 customers is easily de-anonymized through cross referencing with other publicly available datasets.
       url: https://www.wired.com/2007/12/why-anonymous-data-sometimes-isnt/
+- line_id: A.4
+  links:
+    - text: Reports of credit bias in Apple card credit limits spark negative press and investigation.
```

Comment:
I have a bit of a hard time putting this item in this bucket since they had this data, they just didn't check for it so it's really more fairness across groups than lack of data. I'd advocate for putting the credit in fairness across groups and moving the Amazon same delivery service [article](https://www.bloomberg.com/graphics/2016-amazon-same-day/) here instead, which includes this apt quote:

"There is so much systemic bias with respect to race. If you aren’t purposefully trying to identify it and correct it, this bias is likely to creep into your outcomes.”